teacher.rb

```diff
@@ -0,0 +1,11 @@
+require './person'
+class Teacher < Person
+  def initialize(age:, specialization:, name: 'Unknown', parent_permission: true)
+    super(name: name, age: age, parent_permission: parent_permission)
+    @specialization = specialization
+  end
```

Comment:
- [ ] As explained in the above comments, please make the necessary changes to this method so your code doesnt break :+1: